src/gui/MainWindow.cpp

```diff
@@ -409,6 +411,10 @@ void MainWindow::initGui()
 	mSaveAsAction->setIcon(iconLoader->loadForTheme(QLatin1String("saveAs")));
 	connect(mSaveAsAction, &QAction::triggered, this, &MainWindow::saveAsClicked);
 
+	mSaveAllAction->setText(tr("Save All..."));
```

Comment:
I think the convention is to use ellipsis when an additional dialog will be opened, like for save as when you can choose where to save.